zookeeper/core/component.py

```diff
@@ -121,7 +122,7 @@ def _wrap_getattribute(component_cls: Type) -> None:
     means that a priori the `__dict__` of a component instance is empty (of
     non-dunder attributes).
 
-    Field values can come from three sources, in descending order of priority:
+    Field values can come from four sources, in descending order of priority:
       1) A value that was passed into `configure` (e.g. via the CLI), which is
          stored in the `__component_configured_field_values__` dict on the
```

Comment:
Yeah that would be fine as long as that's your intention, but yeah I think it would be confusing if everything you do with setattr implicitly counts as a manual override with the same priority as `configure`, because it would override default values specified in sub components which I don't think would be expected behaviour.